testing_gorums.go

```diff
@@ -37,3 +38,23 @@ func TestSetup(t testing.TB, numServers int, srvFn func(i int) ServerIface) ([]s
 	}
 	return addrs, stopFn
 }
+
+func TestServerSetup(t testing.TB, addr string, srv ServerIface) (func(), func()) {
```

Comment:
Maybe we should not export this func, since I think it is mainly aimed at testing gorums internal logic.